### PR TITLE
ProfileTabs component

### PR DESCRIPTION
**Pull request by Mayowa-Ojo**
Added the `ProfileTabs` component and it's story component.

The profile tabs control which component is rendered below so I assume it will receive props that tell which tab is currently active from the parent component. Also, there will be a controller to update the active tabs in the parent component or wherever the state is managed when they're toggled.

**TODO**:
- [ ] Controller for the tabs selection.

### CODE REVIEW

File: kibbeh/src/modules/room/chat/RoomChatInput.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ export const RoomChatInput: React.FC<ChatInputProps> = ({ users }) => {
           }}
         />
       </div>
-      <div className="flex items-stretch">
+      <div className="flex flex items-stretch">
```
Comment (overlisted) [inline kibbeh/src/modules/room/chat/RoomChatInput.tsx:112]: 🤔

**Review by overlisted — CHANGES_REQUESTED** [review]

File: kibbeh/src/modules/room/chat/RoomChatInput.tsx (hunk 1 above)
Comment (Mayowa-Ojo) [reply kibbeh/src/modules/room/chat/RoomChatInput.tsx]: I just fixed this duplicate class. must have been a mistake

**Comment by overlisted** [comment]
why are you pulling upstream all the time

**Comment by Mayowa-Ojo** [comment]
Err, to keep my fork up to date.

**Comment by overlisted** [comment]
you shouldn't make the pr from staging, make a separate branch for every pr

**Comment by Mayowa-Ojo** [comment]
Alright

**Comment by Mayowa-Ojo** [comment]
I'm closing this PR because I've created another to address the issues you mentioned.

**Pull request closed without merging**